spec/service_newapi.rb

```diff
@@ -119,6 +119,140 @@ def initialize(path)
     dbus_signal :LongTaskEnd
   end
 
+=begin
```

Comment:
This file has leftover notes and ideas. As long as this comment is here, ignore this file.